### ISSUE TITLE
[Device Request] Smart rings (Oura / Motiv)

### ISSUE THREAD

**Issue by 0x53A** [Freeyourgadget/Gadgetbridge#1056]
Hi,

I am interested in getting a smart ring, and there seem to be only two options anyway ([Oura](https://ouraring.com/) and [Motiv](https://mymotiv.com/)).

Before ordering one, I was trying to find any information about whether it is possible to read the data from a third-party app, but couldn't find anything.

__I'd be happy to (well, try to, at least) add support myself, but do you know anything about either device before I buy it?__ It would suck if they encrypted the communication or something.

**Comment by danielegobbetti** [comment]
No, we have no insights unfortunately. The devices look certainly interesting! I wonder if you could motivate either company to publish their protocol (and/or sponsor devices to the floss community 😉) as a competitive advantage over the other one.
